lms/envs/common.py

```diff
@@ -991,6 +991,12 @@
 
 LANGUAGE_DICT = dict(LANGUAGES)
 
+# Languages supported for custom course certificate templates
+CERTIFICATE_TEMPLATE_LANGUAGES = (
+    ('en', u'English'),
+    ('es', u'Español')  # base Spanish
+)
```

Comment:
This should be the default value for `CERTIFICATE_TEMPLATE_LANGUAGES` that is available to all open-source users out of the box. Since we don't know what languages are available to our open-source users, I think it would make sense to just default to 'all-languages'. The edx.org-specific values can be configured in https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml, I believe. Check with devops to be sure.

Reply:
So the ALL_LANGUAGES list that we have in common.py does not include the language names translated into their native language, and the LANGUAGES list that we have -- in which language names are listed in their native language -- contains fake languages and language codes that are more than two letters.
Do you think it would make sense to create a new list that is a subset of LANGUAGES that contains only real languages with codes that are two letters, or use ALL_LANGUAGES even though the names are all in english?

Reply:
What I meant was that the default should just be the empty `'All Languages'` item I mentioned in another comment.

Reply:
So they'd just be one choice by default, 'All Languages', unless the site language configured a custom list of languages in their site-specific configuration.

Reply:
Oh right yes, that makes sense. Thank you!